### DISCUSSION TITLE
Define array with list

### DISCUSSION THREAD

**Discussion by pluja** [pydantic/pydantic#4644] [Question] [answered]
I have the following dictionary structure:

```
{
        "username": "username",
        "preferences": {
            "first": [],
            "second": [],
            "third": []
         },
         "last_accessed": "",
         "last_interaction_id": "",
         "last_check": "2022-05-10T16:13:41.883637"
}
```

How could I translate the `preferences` dictionary in a pydantic class? I have tested the following but I am getting an error

```
class User(BaseModel):
    id: int
    username = str
    preferences = Dict[str, list[str]]
    last_accessed: datetime | None = None
    last_check: datetime | None = None
    last_interaction_id: int = 0
```

The error: `RuntimeError: no validator found for <class 'typing._GenericAlias'>, see 'arbitrary_types_allowed' in Config`

Thank you!

**Comment by PrettyWood** [comment, marked as answer]
Hi @pluja 
It's because you used `username = str` instead of `username: str` and same for your dict

**Reply by pluja** [reply]
Oh, sorry. Did not notice that. Thank you.
